### PR TITLE
Gwh apidocs 1739 opt out api

### PR DESCRIPTION
**Pull request by genwhittTTD**
Queries resolved and edits addressed.
Updated diagram is in
Note I copied the one new file into the Japanese file set, and updated the diagram in the Japanese. Otherwise, Japanese updates are pending translation and will not be in this PR.

### CODE REVIEW

File: docs/endpoints/post-optout-status.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    "ufv1uGRovNiJNbJqiE/xzM+aKE7jP69MgspOZoEQ3xc=",
+    "zstfu9RG/Ih5trR6hlaHP6hw5yt9mYd5TKg2mRpWVng=",
+    "VZqcve81+ImeoNUsizTFLEMOvREptJo1ROZvKw9ibSM="
+  ]
+}
+```
+
+The following is an encrypted opt-out status request example:
+
+(**GWH_MC just checking that the example below is all correct, with the pipe and the links in it?**)
```
Comment (mcollins-ttd) [inline docs/endpoints/post-optout-status.md:70]: Yes, looks correct to me.

File: docs/endpoints/summary-endpoints.md [hunk 2]
```diff
@@ -22,12 +22,13 @@ The following endpoints are for retrieving and managing UID2 tokens (identity to
 | [POST&nbsp;/token/validate](post-token-validate.md) | Used for testing, to validate that an advertising token (UID2) matches the specified email address, phone number, or the respective hash. | Required | Required |
 | [POST&nbsp;/token/refresh](post-token-refresh.md) | Generate a new token for a user for their refresh token from the [POST&nbsp;/token/generate](post-token-generate.md) response. | N/A | Required |
 
-## Identity Maps
+## Identity Map and Opt-Out
 
 The following endpoints are used by advertisers and third-party data providers. Publishers do not need to use these endpoints.
```
Comment (mcollins-ttd) [inline docs/endpoints/summary-endpoints.md:27]: If we add opt-out to this section, we might need to update the text as /optout/status is also for DSPs and sharers.

Alternatively we could put opt-out in its own section. I'm good either way.

File: docs/endpoints/post-optout-status.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+import Link from '@docusaurus/Link';
+
+# POST /optout/status
+
+Checks the opt-out status of raw UID2s. Given a list of raw UID2s, this endpoint returns the raw UID2s that have opted out, as well as the time that the opt-out took place. For more information, see [User Opt-Out](../getting-started/gs-opt-out.md).
+
+Used by: This endpoint is used by advertisers, data providers, DSPs, and sharers. More generally, this is for participants who have access to raw UID2s but do not have access to the underlying email addresses or phone numbers and want to know opt-out status.
+
+For details, see one of the following guides:
```
Comment (sunnywu) [inline docs/endpoints/post-optout-status.md:16]: should this line be reworded to include docs/sharing/sharing-overview for sharers? (the paragraph aboves mentions advertisers/DP/dsp/sharer but only point to integ guide for DSP/DP/advertiser?

Comment (genwhittTTD) [reply docs/endpoints/post-optout-status.md:16]: Great call Sunny. Done. Thx.

File: docs/endpoints/post-optout-status.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/endpoints/post-optout-status.md:70]: Awesome, thx!

File: docs/endpoints/summary-endpoints.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/endpoints/summary-endpoints.md:27]: @mcollins-ttd very good point, thx. Making it into a new section as you originally had it.

**Review by sunnywu — APPROVED** [review]

**Review by mcollins-ttd — APPROVED** [review]

File: docs/endpoints/post-optout-status.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+:::note
+The integration environment and the production environment require different <Link href="../ref-info/glossary-uid#gl-api-key">API keys</Link>.
+:::
+
+### Unencrypted JSON Body Parameters
+
+There is a single body parameter.
+
+| Body Parameter | Data Type | Attribute | Description | Format |
```
Comment (asloobq) [inline docs/endpoints/post-optout-status.md:51]: "Format" is an empty column here. Probably better to remove it ?

Comment (genwhittTTD) [reply docs/endpoints/post-optout-status.md:51]: Fixed in follow-up PR. Thanks!

**Pull request merged**